packages/psss/src/components/DateToolbar/DateToolbar.js

```diff
@@ -4,43 +4,50 @@
  */
```

Comment:
@kael89 Sorry I made some last minute changes to the `DateToolbar` this morning because when I was testing I noticed some inconsistencies (UTC vs NZT) with the dates I was getting from date-fns and the tupaia period utils. My fix is to update everywhere to use the utls and now it works well. I should have listened to you the first time when you said to encapsulate the week formatting in tupaia utls 😄